subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/webserver/webserver.go

```diff
@@ -79,6 +79,9 @@ func NewWebServer(sConfig *si.Config,
 	ws.route(si.BindPath, ws.serve(ws.serveBindPath))
 	ws.route(si.PreemptPath, ws.serve(ws.servePreemptPath))
 	ws.route(si.AffinityGroupsPath, ws.serve(ws.serveAffinityGroups))
+	ws.route(si.ClusterStatusPath, ws.serve(ws.serveClusterStatus))
+	ws.route(si.PCStatusPath, ws.serve(ws.servePCStatus))
+	ws.route(si.VCsStatusPath, ws.serve(ws.serveVCsStatus))
```

Comment:
As they are more specific routers, can we work as expected even we did not move they above:
 ws.route(si.ClusterStatusPath, ws.serve(ws.serveClusterStatus))
?